src/providers/aihorde/aihorde.go

```
@@ -0,0 +1,193 @@
+
+	apiKey := "0000000000"
+	if params.ApiKey != "" {
+		apiKey = params.ApiKey
+	} else if envKey := os.Getenv("AIHORDE_API_KEY"); envKey != "" {
+		apiKey = envKey
+	} else if envKey := os.Getenv("AI_API_KEY"); envKey != "" {
+		apiKey = envKey
+	}
+
+	// Build prompt with chat format
+	prompt := input
```

golangci-lint (2.12.2)

[error] ineffectual assignment to prompt

(ineffassign)